src/content/vagrant/product-landing.json

```
@@ -14,72 +14,151 @@
 	},
 	"get_started": {
 		"heading": "Getting Started",
-		"body": "Use Vagrant to securely store, access, and manage secrets and other sensitive data.",
+		"body": "Use Vagrant to create and configure a reproducible and portable work environment.",
 		"ctas": [
 			{
 				"text": "CLI Quick Start",
-				"url": "/vault/tutorials/getting-started"
-			},
-			{
-				"text": "CLI Quick Start",
-				"url": "/vault/tutorials/cloud"
+				"url": "/vagrant/tutorials/getting-started"
 			}
 		]
 	},
 	"blocks": [
 		{
 			"type": "heading",
-			"heading": "Use Cases",
+			"heading": "Fundamentals",
 			"level": 2,
 			"size": 400
 		},
 		{
-			"type": "tutorial_cards",
-			"tutorialSlugs": ["vault/transform", "vault/static-secrets"]
+			"type": "linked_cards",
+			"cards": [
+				{
+					"heading": "Boxes",
+					"body": "The package format for Vagrant environments - create your own or use one from the public catalog.",
+					"url": "/vagrant/docs/boxes"
+				},
+				{
+					"heading": "Install and Specify a Box",
+					"body": "Use a base image to quickly clone a virtual machine.",
+					"url": "/vagrant/getting-started-boxes"
```

```suggestion
					"url": "/vagrant/tutorials/getting-started/getting-started-boxes"
```
Needs the full path to the tutorial in a collection.